ci/glci/openstack_image.py

```diff
@@ -0,0 +1,96 @@
+import dataclasses
+
+from datetime import datetime
+from time import sleep
+
+import glci
+
+from openstack import connect
```

Comment:
nit: by convention, imports should be grouped, like so:

```
<imports from std python> # e.g. dataclasses, datertime, time, ..

<imports from other 3rdparty> # e.g. requests, openstack, ..

<imports from local> # e.g. glci
```
see https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Done. Connection is now also cached.